qtconsole/frontend_widget.py

```diff
@@ -169,19 +168,10 @@ def _lexer_default(self):
 
     def __init__(self, local_kernel=_local_kernel, *args, **kw):
         super(FrontendWidget, self).__init__(*args, **kw)
-        # FIXME: remove this when PySide min version is updated past 1.0.7
-        # forcefully disable calltips if PySide is < 1.0.7, because they crash
```

Comment:
@ccordoba12 this is why I think I killed PySide support.

Reply:
I guess it is reasonable to expect people not to be using a version from 2011??

https://pypi.org/project/PySide/

Reply:
Agreed.